lms/views/api/canvas/authorize.py

```diff
@@ -31,6 +31,16 @@
 )
 
 
+#: The Canvas API scopes that we need for our groups feature.
+GROUPS_SCOPES = (
+    "url:GET|/api/v1/courses/:course_id/groups",
```

Comment:
Again, more that we need here, will clean as I clean the methods on canvas/client